### DISCUSSION TITLE
Please provide your linked in Usernames to post reg project.

### DISCUSSION THREAD

**Discussion by Devamani11D** [Devamani11D/salt-app-kickstart#85] [Announcements]
Heyaa folks !!🧑‍🤝‍🧑 
@adityakalburgi @ananyag309 @milansamuel609 @PrityanshuSingh @Ayush07571 @samyak-aditya @nikhilkalburgi @AnujShrivastava01 @devkumar2313 @jjf2009 @shubhagarwal1 @RamharshDandekar @T-Rahul-prabhu-38

Let's connect and grow together by sharing knowledge and resources.
Please provide your linked in usernames so that it will be easy to tag you on linkedin when posting about the project.
Thank you!!
🚀

**Comment by AnujShrivastava01** [comment]
> Heyaa folks !!🧑‍🤝‍🧑 
> @adityakalburgi @ananyag309 @milansamuel609 @PrityanshuSingh @Ayush07571 @samyak-aditya @nikhilkalburgi @AnujShrivastava01 @devkumar2313 @jjf2009 @shubhagarwal1 @RamharshDandekar @T-Rahul-prabhu-38
> 
> Let's connect and grow together by sharing knowledge and resources.
> Please provide your linked in usernames so that it will be easy to tag you on linkedin when posting about the project.
> Thank you!!
> 🚀 


- AnujShrivastava01
https://www.linkedin.com/in/anuj-shrivastava-949645299?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app

Username : Anuj Shrivastava

**Comment by jjf2009** [comment]
> Heyaa folks !!🧑‍🤝‍🧑 @adityakalburgi @ananyag309 @milansamuel609 @PrityanshuSingh @Ayush07571 @samyak-aditya @nikhilkalburgi @AnujShrivastava01 @devkumar2313 @jjf2009 @shubhagarwal1 @RamharshDandekar @T-Rahul-prabhu-38
> 
> Let's connect and grow together by sharing knowledge and resources. Please provide your linked in usernames so that it will be easy to tag you on linkedin when posting about the project. Thank you!! 🚀
.Jared Furtado 
https://www.linkedin.com/in/jared-furtado/

**Reply by AnujShrivastava01** [reply]
https://www.linkedin.com/in/anuj-shrivastava-949645299?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app

Username: Anuj Shrivastava

**Comment by milansamuel609** [comment]
> Heyaa folks !!🧑‍🤝‍🧑 @adityakalburgi @ananyag309 @milansamuel609 @PrityanshuSingh @Ayush07571 @samyak-aditya @nikhilkalburgi @AnujShrivastava01 @devkumar2313 @jjf2009 @shubhagarwal1 @RamharshDandekar @T-Rahul-prabhu-38
> 
> Let's connect and grow together by sharing knowledge and resources. Please provide your linked in usernames so that it will be easy to tag you on linkedin when posting about the project. Thank you!! 🚀

https://www.linkedin.com/in/milan-p-samuel-b54290242

**Reply by AnujShrivastava01** [reply]
https://www.linkedin.com/in/anuj-shrivastava-949645299?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app

Username: Anuj Shrivastava

**Comment by PrityanshuSingh** [comment]
Hey Devamani11D, it was great working with you, looking forward to work on
similar projects in future whenever time permits.
Here is my LinkedIn Profile : https://www.linkedin.com/in/prityanshu-singh

**Comment by trahulprabhu38** [comment]
hello , this is my linkedin : www.linkedin.com/in/trahulprabhu38

**Comment by adityakalburgi** [comment]
> Heyaa folks !!🧑‍🤝‍🧑 @adityakalburgi @ananyag309 @milansamuel609 @PrityanshuSingh @Ayush07571 @samyak-aditya @nikhilkalburgi @AnujShrivastava01 @devkumar2313 @jjf2009 @shubhagarwal1 @RamharshDandekar @T-Rahul-prabhu-38
> 
> Let's connect and grow together by sharing knowledge and resources. Please provide your linked in usernames so that it will be easy to tag you on linkedin when posting about the project. Thank you!! 🚀

Hi @Devamani11D here is my LinkedIn profile link
https://www.linkedin.com/in/adityakalburgi

**Comment by Ayush07571** [comment]
hello @Devamani11D, here is my Linkedin profile : Ayush Karan ( www.linkedin.com/in/ayush-karan )

**Comment by Ramharsh-aidev** [comment]
> Heyaa folks !!🧑‍🤝‍🧑 @adityakalburgi @ananyag309 @milansamuel609 @PrityanshuSingh @Ayush07571 @samyak-aditya @nikhilkalburgi @AnujShrivastava01 @devkumar2313 @jjf2009 @shubhagarwal1 @RamharshDandekar @T-Rahul-prabhu-38
> 
> Let's connect and grow together by sharing knowledge and resources. Please provide your linked in usernames so that it will be easy to tag you on linkedin when posting about the project. Thank you!! 🚀

Hello @Devamani11D, this is my LinkedIn profile link
[https://www.linkedin.com/in/ramharsh-sanjay-dandekar](url)

**Comment by ananyag309** [comment]
> Heyaa folks !!🧑‍🤝‍🧑 @adityakalburgi @ananyag309 @milansamuel609 @PrityanshuSingh @Ayush07571 @samyak-aditya @nikhilkalburgi @AnujShrivastava01 @devkumar2313 @jjf2009 @shubhagarwal1 @RamharshDandekar @T-Rahul-prabhu-38
> 
> Let's connect and grow together by sharing knowledge and resources. Please provide your linked in usernames so that it will be easy to tag you on linkedin when posting about the project. Thank you!! 🚀

https://www.linkedin.com/in/ananya-gupta-30aa9b28b?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app

**Comment by shubhagarwal1** [comment]
> Heyaa folks !!🧑‍🤝‍🧑 @adityakalburgi @ananyag309 @milansamuel609 @PrityanshuSingh @Ayush07571 @samyak-aditya @nikhilkalburgi @AnujShrivastava01 @devkumar2313 @jjf2009 @shubhagarwal1 @RamharshDandekar @T-Rahul-prabhu-38
> 
> Let's connect and grow together by sharing knowledge and resources. Please provide your linked in usernames so that it will be easy to tag you on linkedin when posting about the project. Thank you!! 🚀

Hey @Devamani11D, sounds great! It’s been a rewarding experience working on this project—especially developing a reusable component from scratch without relying on external libraries, ensuring lightweight performance and a personalized user experience. This project has been a fantastic learning opportunity. Here’s my LinkedIn profile: https://www.linkedin.com/in/shubh-agarwal1. Looking forward to connecting and sharing insights!

**Comment by nikhilkalburgi** [comment]
Hi @Devamani11D, this is a fantastic initiative—congratulations on making it happen! I truly enjoyed exploring and mentoring on this project. A big thank you to everyone for their valuable contributions. Feel free to connect with me here: https://linkedin.com/in/nikhilkalburgi.

**Comment by samyak-aditya** [comment]
> Heyaa folks !!🧑‍🤝‍🧑 
> @adityakalburgi @ananyag309 @milansamuel609 @PrityanshuSingh @Ayush07571 @samyak-aditya @nikhilkalburgi @AnujShrivastava01 @devkumar2313 @jjf2009 @shubhagarwal1 @RamharshDandekar @T-Rahul-prabhu-38
> 
> Let's connect and grow together by sharing knowledge and resources.
> Please provide your linked in usernames so that it will be easy to tag you on linkedin when posting about the project.
> Thank you!!
> 🚀 

https://www.linkedin.com/in/samyak-aditya
